src/document-state.js

```diff
@@ -112,6 +116,32 @@ export class DocumentState {
   onVisibilityChanged_() {
     this.visibilityObservable_.fire();
   }
+
+  /**
+   * @param {function()} handler
```

Comment:
[Z̧̡̩̞͕̘̳̗͖̱̉͋͛̆̄͋͆͊̕͠a̢̢̟͎̬͕̭̻͎͙̋͗̾̈́̂̀̊̽͑͘l̨̧̢̰̦̥͎̠̥̀͐̈̎̍̓͌̆̋͜͝g̢̡̘̮͙̞̟̭̹̳̀̅̓͗̐͑̀̕͝͝o̯̻͙̦̱̘̮̰̣̎̃͛͆̅̽͌̽͐̃͜](http://blog.izs.me/post/59142742143/designing-apis-for-asynchrony). Unfortunately, his dark highness found our codebase.